testing/kdc.json

```diff
@@ -1 +0,0 @@
-{"id": "kdc", "instances": 1, "cpus": 1, "mem": 512, "container": {"type": "MESOS", "docker": {"image": "mesosphere/kdc:latest", "forcePullImage": true}, "portMappings": [{"containerPort": 88, "protocol": "tcp", "labels": {"VIP_0": "kdc:88"}}, {"containerPort": 88, "protocol": "udp", "labels": {"VIP_0": "kdc:88"}}, {"containerPort": 8000, "protocol": "tcp", "labels": {"VIP_0": "kdc:8000"}}]}, "networks": [{"mode": "container/bridge"}]}
```

Comment:
Is this supposed to be deleted?

Reply:
Yes, we refer to `tools/kdc.json` in our deployment, and this was an extra file included erroneously in #1824